src/wallet/wallet.cpp

```diff
@@ -4535,7 +4535,7 @@ util::Result<MigrationResult> MigrateLegacyToDescriptor(std::shared_ptr<CWallet>
         if (!local_wallet->MigrateToSQLite(error)) return util::Error{error};
 
         // Do the migration of keys and scripts for non-blank wallets, and cleanup if it fails
```

Comment:
The comment can be updated now to account for empty wallets as well.

Reply:
I'd just keep "for non-empty wallets" instead, as we did remove the flag check as well in a previous iteration.